README.md

```diff
@@ -60,12 +60,14 @@ docker run --name software-guide -p 8888:8888 insighttoolkit/itksoftwareguide-ed
 ```sh
 ssh-keygen
 cat ~/.ssh/id_rsa.pub
-# Paste this value into https://github.com/settings/keys cd /ITKSoftwareGuide ./Utilities/SetupForDevelopment.sh
+# Paste this value into https://github.com/settings/keys
+cd /ITKSoftwareGuide
+./Utilities/SetupForDevelopment.sh
 cd /ITKSoftwareGuide-build/ITKSoftwareGuide-build
 ninja
 ```
-  6. To view the built PDF's, navigate to `bin/ITKSoftwareGuide/SoftwareGuide/Latex/`,
-     click on the PDF, then the *download* button.
+  6. To view the built PDF's, navigate to `/ITKSoftwareGuide-build/ITKSoftwareGuide-build/SoftwareGuide/Latex/`,
```

Comment:
**Repeating the string `/ITKSoftwareGuide-build`** looks suspicious to me, i.e. at first sight I'd think of it as an oversight. **Although the first one means the local build directory** (I ignore whether this is hard-coded in the Docker image), it is still **weird**.

**We have used**, for example
```
\code{<ITK build directory>/lib/cmake/\ITKVERSIONMAJORMINOR/Modules/}
```
in 
```
SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex
```

I **sought for** `build/` or `bin/` **across the repository** for an indication of the ITK SWG build directory, but found **0** relevant (`*.md`, or `*.tex`) **matches**.

In **ITK itself, **we have used**
```
../ITK-build
```
in
```
Documentation/Data.md
```

I'm open to suggestions to **standardize this, if possible**.

Reply:
Thanks for the review, @jhlegarreta !

Yes, `/ITKSoftwareGuide-build/ITKSoftwareGuide-build` is what was intended. This is a result of the Superbuild -- `/ITKSoftwareGuide-build/ITKSoftwareGuide-build` sits along side `/ITKSoftwareGuide/ITK-build/`. The "Superbuild" first builds ITK,  then it builds ITKSoftwareGuide against ITK.

Reply:
This is also a standard layout for other Superbuild projects, such as `ITKExamples` or `Slicer`.

Reply:
OK. As for standardizing the build directory naming in the docs
```
ITK-build
```
(and ```ITK-sources``` for example, and their variants for the SW Guide, examples, etc.) seem appropriate to me.

Reply:
Yes, good point -- we should use `ITK-build` consistently in the docs when possible. Since the default directory name when cloning the source repository is `ITK`, I think that is a good name to standardize on for the source code directory.